src/components/TaxPayer/PersonFields.tsx

```diff
@@ -15,36 +16,40 @@ interface PersonFieldsProps<T extends Person> extends BaseFormProps {
   defaults?: T
   children?: ReactNode
   errors: DeepMap<Partial<Person>, FieldError>
+  person?: Person | null | undefined
 }
 
-export const PersonFields = <T extends Person>({ register, errors, defaults, children }: PersonFieldsProps<T>): ReactElement => (
+export const PersonFields = <T extends Person>({ register, errors, defaults, children, person }: PersonFieldsProps<T>): ReactElement => (
   <div>
     <LabeledInput
       label="First Name and Initial"
+      aria-label="First Name and Initial"
       register={register}
       name="firstName"
       patternConfig={Patterns.name}
       required={true}
       error={errors.firstName}
-      defaultValue={defaults?.firstName}
+      defaultValue={(person !== null) ? person?.firstName : defaults?.firstName}
```

Comment:
the defaultValue property is allowed to be undefined. I don't think there's a reason for a three case value here.

I think it could just be: `person?.firstName ?? defaults?.firstname`, as the `??` operator allows the right side expression if the left side expression is null or undefined.

Reply:
Good suggestion! I haven't used the nullish coalescing operator before, glad to get to put it to use.